src/DataViews/PresetsDataView.cpp

```diff
@@ -191,8 +191,8 @@ DataView::ActionStatus PresetsDataView::GetActionStatus(std::string_view action,
 
   if (action == kMenuActionDeletePreset || action == kMenuActionShowInExplorer) {
     return ActionStatus::kVisibleAndEnabled;
-
-  } else if (action == kMenuActionLoadPreset) {
+  }
+  if (action == kMenuActionLoadPreset) {
```

Comment:
The else below should also be removed then, otherwise this creates an inconsistency that makes the code more confusing.

Reply:
Yeah, maybe clang-tidy only shows you a warning for the first else in the chain and so you would have to apply it repeatedly.